test/unit/TokenController/totalBalanceOf.js

```diff
@@ -0,0 +1,155 @@
+    const [member1] = this.accounts.members;
+    this.nxmBalanceBefore = await nxm.balanceOf(member1.address);
+  });
+
+  it('should return 0 if address has no balance', async function () {
+    const { tokenController } = this.contracts;
+    const [nonMember1] = this.accounts.nonMembers;
+
+    expect(await tokenController.totalBalanceOf(nonMember1.address)).to.equal(0);
+    expect(await tokenController.totalBalanceOfWithoutDelegations(nonMember1.address)).to.equal(0);
+  });
+
+  it('should correctly calculate simple balance', async function () {
+    const { tokenController, nxm } = this.contracts;
+    const [internalContract] = this.accounts.internalContracts;
+    const [member1] = this.accounts.members;
+
+    // Mint 1
+    const amount = One;
+    await tokenController.connect(internalContract).mint(member1.address, amount);
+
+    // [balanceOf, totalBalanceOf, totalBalanceOfWithoutDelegations] should all return the same result
+    expect(await tokenController.totalBalanceOf(member1.address)).to.equal(amount.add(this.nxmBalanceBefore));
+    expect(await tokenController.totalBalanceOfWithoutDelegations(member1.address)).to.equal(
+      await nxm.balanceOf(member1.address),
+    );
+  });
+
+  it('should correctly calculate staker rewards', async function () {
+    const { tokenController, pooledStaking, nxm } = this.contracts;
+    const [member1] = this.accounts.members;
+
+    // mock staker rewards
+    const amount = One;
+    await pooledStaking.setStakerReward(member1.address, amount);
+
+    expect(await tokenController.totalBalanceOf(member1.address)).to.equal(amount.add(this.nxmBalanceBefore));
+    expect(await tokenController.totalBalanceOfWithoutDelegations(member1.address)).to.equal(
+      amount.add(await nxm.balanceOf(member1.address)),
+    );
+  });
+
+  it('should correctly calculate staker deposits', async function () {
+    const { tokenController, pooledStaking } = this.contracts;
+    const [member1] = this.accounts.members;
+
+    // mock staker deposits
+    const amount = One;
+    await pooledStaking.setStakerDeposit(member1.address, amount);
+
+    expect(await tokenController.totalBalanceOf(member1.address)).to.equal(amount.add(this.nxmBalanceBefore));
+    expect(await tokenController.totalBalanceOfWithoutDelegations(member1.address)).to.equal(
+      await tokenController.totalBalanceOf(member1.address),
+    );
+  });
+
+  it('should correctly calculate assessment stake', async function () {
+    const { assessment, tokenController } = this.contracts;
+    const [member1] = this.accounts.members;
+
+    const amount = One.mul(2).pow(80);
+    await assessment.setStakeOf(member1.address, amount);
+
+    expect(await tokenController.totalBalanceOf(member1.address)).to.equal(amount.add(this.nxmBalanceBefore));
+    expect(await tokenController.totalBalanceOfWithoutDelegations(member1.address)).to.equal(
+      await tokenController.totalBalanceOf(member1.address),
+    );
```

Comment:
```
    const amount = One.mul(2).pow(80);
    await assessment.setStakeOf(member1.address, amount);

    const expecedAmount = this.nxmBalanceBefore.add(amount);

    expect(await tokenController.totalBalanceOf(member1.address)).to.equal(expecedAmount);
    expect(await tokenController.totalBalanceOfWithoutDelegations(member1.address)).to.equal(expecedAmount);
```

Reply:
This pattern seems to be repeated in a few places. The suggested change is easier to read.